docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/keeping-cohesion-with-unified/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Attribution"
+title: "Keeping cohesion between the Unified and Attribution packages"
 description: "Keeping cohesion between the Unified and Attribution packages"
 sidebar_position: 10
 hide_title: true
```

Comment:
This page doesn't have a title showing under the description

![Screenshot 2024-11-29 at 09 42 34](https://github.com/user-attachments/assets/351ac9a1-5653-4b36-a2a6-622f6e0a2288)